rosdep/base.yaml

```diff
@@ -1290,12 +1294,12 @@ libcap-dev:
   arch: [libcap]
   debian: [libcap-dev]
   fedora: [libcap-devel]
-  gentoo: [sys-libs/libcap]
+  gentoo: ['=sys-libs/libcap-2*']
```

Comment:
We must be coherent. In debian -dev lose the version number
You can see here https://packages.debian.org/sid/libcap-dev came from the same source as libcap2